src/components/LogUpdate/CriticalCarePreview.tsx

```diff
@@ -49,14 +52,27 @@ export default function CriticalCarePreview(props: Props) {
 
   return (
     <div className="w-full transition-all duration-200 ease-in-out md:mx-auto md:max-w-5xl md:pt-8">
-      <div className="py-4">
+      <div className="py-4 flex items-center justify-between max-[368px]:flex-col max-[368px]:items-start">
         <ButtonV2
           id="back-to-consultation"
           variant="secondary"
           href={`/facility/${props.facilityId}/patient/${props.patientId}/consultation/${props.consultationId}`}
         >
           {t("back_to_consultation")}
         </ButtonV2>
+        <Link
+          className="mt-2"
+          href={`/facility/${props.facilityId}/patient/${props.patientId}/consultation/${props.consultationId}/log_updates/${props.id}/update`}
+        >
+          <Button
+            id="update-log"
+            variant={"primary"}
+            data-testid="update-log-button"
+            aria-label={t("update_log")}
+          >
+            {t("update_log")}
+          </Button>
+        </Link>
```

Comment:
This right way to compose links using button is as shown.

Eg. usage in care: https://github.com/ohcnetwork/care_fe/blob/532945766d2b76bd64dc4ce7ced9aceedece313d/src/components/Common/Breadcrumbs.tsx#L77-L83

See shadcn's docs: https://ui.shadcn.com/docs/components/button#as-child